cdap-data-fabric/src/test/java/co/cask/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableTest.java

```diff
@@ -106,10 +106,11 @@ public void testTablesSurviveAcrossRestart() throws Exception {
       Assert.assertTrue(admin.exists());
     }
 
-    // clearing in-mem cache - mimicing JVM restart
+    // clearing in-mem cache - mimicking JVM restart
     service.clearTables();
     for (String tableName : tableNames) {
-      service.list().contains(tableName);
+      Preconditions.checkState(service.list().contains(
+        PrefixedNamespaces.namespace(cConf, CONTEXT1.getNamespaceId(), tableName)));
     }
   }
 }
```

Comment:
My previous comment I meant that this test should have an @After clause that calls levelDBTableService.close(). And same for the other tests.

Reply:
Since the service is like a singleton object, I don't want to call close, which makes the service unusable after closing it.
`clearTables` is very similar.